tests/telemetry_derived/clients_daily_v7/test_aggregation/main_summary_v4.yaml

```diff
@@ -0,0 +1,298 @@
+---
+- &base
+  client_id: test
+  document_id: base
+  sample_id: 0
+  submission_date: '2019-01-01'
+- <<: *base
```

Comment:
I refactored this test input to yaml because it's unreadable as json, and yaml allows anchors and references for reducing redundancy